src/main/java/api/apiwrappers/PetsApiWrapper.java

```diff
@@ -0,0 +1,61 @@
+package api.apiwrappers;
+
+import java.io.IOException;
+import api.CommonApiWrapper;
+import api.domain.pets.*;
+
+public class PetsApiWrapper extends CommonApiWrapper{
+    public Pet postPet(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet";
+
+        return (Pet)post(pet, endpoint);
```

Comment:
We should handle here the IOException from post method, so it's not passed further into the test method

Reply:
I'm not sure what I'm supposed yo do here.

Reply:
Added TryCatch exception